CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## HEAD
 
+* Added contexts feature. Thanks to @apurvis.
```

Comment:
I don't always know if the next version will be major, minor, or patch until I know everything going into it. So I keep it HEAD until I cut the version.